src/main/java/uk/gov/hmcts/ccd/infrastructure/user/UserAuthorisationFactory.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.ccd.infrastructure.user;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.context.annotation.RequestScope;
+import uk.gov.hmcts.ccd.domain.service.common.CaseAccessService;
+import uk.gov.hmcts.ccd.infrastructure.user.UserAuthorisation.AccessLevel;
+import uk.gov.hmcts.reform.auth.checker.spring.serviceanduser.ServiceAndUserDetails;
+
+@Configuration
+public class UserAuthorisationFactory {
```

Comment:
can we rename this to make it more evident that this is a spring configuration class?

Reply:
👍